2023/refractive-index-questions/french/sentence_translations.json

```diff
@@ -101,7 +101,7 @@
  {
   "input": "This particular animation is a little bit messy and hard to follow, so it might be clearer if instead we simply color every point of the plane, such that those points are white near the crests of the wave, and then black away from the crests.",
   "model": "nmt",
-  "translatedText": "Cette animation est un peu brouillonne et difficile à suivre, donc pour gagner en clarté nous colorons simplement chaque point du plan, de telle sorte qu'un point soit blanc près des crêtes de l'onde, et noir à l'écart des crêtes.",
+  "translatedText": "Cette animation en particulier est un peu chargée et difficile à suivre, alors ce serait être plus claire si nous colorions plutôt simplement chaque point du plan, de telle sorte qu'ils soient blancs près des crêtes de la vague, puis noirs à l'écart des crêtes.",
```

Comment:
"onde" is the appropriate translation for "wave" in this context, not "vague"

Reply:
You're right. I did the correction elsewhere, but missed this one.